### PR TITLE
Created card(s) routes, model and controller

### PR DESCRIPTION
**Pull request by Salaz-art**
Created the Card model, which belongs to Users.

Created new routes for cards: index, show, new, create, destroy (didn't add edit & update as we didn't specify them in our user stories doc). 

Created a Cards Controller with definition for the actions stated above - I already added basic code for them to work once they will be called, which will need to be tailored as we grow the app.

Added Gem faker and seeded the DB with 5 fake users that each have 2 fake cards (same for all users atm).

Created the basis of our Pokemon Cards page (index view) which displays cards title, description, ppd and vendor.
Created show view.
Created new view.

Did 2 migrations: 
- 1 to create a cards table with columns title, description, PPD and foreign key to Users.
- 1 to add columns to the users table: first_name, last_name, username, payment_details.

### CODE REVIEW

**Review by Madeleine-dk — COMMENTED** [review]
omg that looks dope ! get it queen but i did not have time to look at your code.

File: app/controllers/cards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    @card = Card.find(params[:id])
+  end
+
+  def new
+    @card = Card.new
+  end
+
+  def create
+    @card = Card.new(card_params)
+    @card.save
```
Comment (Salaz-art) [inline app/controllers/cards_controller.rb]: You're trying to create a card without associating it to a user.
Idially before` .save` you want to do the associattion

Comment (Salaz-art) [inline app/controllers/cards_controller.rb]:
```suggestion
    @card.user = current_user
    @card.save
```

File: app/controllers/cards_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    redirect_to card_path
```
Comment (Salaz-art) [inline app/controllers/cards_controller.rb]: Don't forget to do a condition if save redirecty otherwise render again

File: app/views/cards/index.html.erb
Comment (Salaz-art) [inline app/views/cards/index.html.erb:1]: Please dont forget to indent things inside a block 🙏 😘

**Comment by Salaz-art** [comment]
Approved BY Pedro-Pan

**Pull request merged**